spec/support/capybara.rb

```diff
@@ -1,3 +1,5 @@
+Webdrivers::Chromedriver.required_version = "114.0.5735.90"
```

Comment:
Za mna vsetko ok. Povedz este @jsuchal ci si s tymto ok, ze sa sem prida verzia.

Reply:
Myslim, ze dohoda s hankou bola, ze to hlavne chceme napisat na hlavnu stranku af a dh - https://ekosystem.slovensko.digital/sluzby/datahub a https://ekosystem.slovensko.digital/sluzby/autoform ak si spravne pamatam.

Reply:
Ja som si ešte písal s Hankou a povedala, že stači pridať ten odsek, ale ešte sa teda uisťím s ňou.